src/Shared/EmptyListWrapper.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react'
+import { View, Text, StyleSheet, Image } from 'react-native'
+import Icon from 'react-native-vector-icons/dist/MaterialIcons'
+
+import WrappedTextButton from '../TextButton/TextButton.js'
+
+export default class EmptyListWrapper extends Component {
+  render() {
+    let { listEmptyState, items, children } = this.props
+
+    if (!listEmptyState) {
+      return <>{children}</>
+    }
+
+    let {
+      buttonType,
+      buttonText,
+      buttonIcon,
+      buttonPrimaryColor,
+      buttonContrastColor,
+      buttonBorderRadius,
+      buttonShadow,
+      buttonUpperCase,
+      buttonAction,
+      buttonWidth,
+    } = listEmptyState
+
+    if (!items || items.length === 0) {
```

Comment:
```suggestion
    if (items && items.length === 0) {
```
You actually specifically want to check for when there is an array that just doesn't have any items